packages/common/package.json

```diff
@@ -36,7 +36,7 @@
     "prom-client": "^14.2.0"
   },
   "dependencies": {
-    "cross-fetch": "3.2.0-alpha.2",
+    "cross-fetch": "4.0.0",
```

Comment:
I think we should be able to remove this dependency, since node v18 comes with fetch built-in